src/factories/requestFactory.ts

```diff
@@ -0,0 +1,61 @@
+import { v4 as uuidv4 } from "uuid";
+
+import { NewPropertyFormData } from "../components/new-asset-form/schema";
+import { managingOrganisations } from "../utils/managing-organisations";
+
+import { CreateNewAssetRequest } from "@mtfh/common/lib/api/asset/v1";
+
+export const assetToCreateAssetAddressRequest = (values: NewPropertyFormData) => {
+  const parentAssetIds: string[] = [];
+  if (values?.propertyEstate && values.propertyEstate !== "")
+    parentAssetIds.push(values.propertyEstate);
+  if (values?.propertyBlock && values.propertyBlock !== "")
+    parentAssetIds.push(values.propertyBlock);
+  if (values?.propertySubBlock && values.propertySubBlock !== "")
+    parentAssetIds.push(values.propertySubBlock);
```

Comment:
Again, I know I wrote this, but this can probably be extracted into a separate method